servicetalk-http-api/src/test/java/io/servicetalk/http/api/ContentEncodingHttpServiceFilterTest.java

```diff
@@ -88,45 +87,35 @@ void testMatchAndRemoveEncodingMiddleDoesNotMatch() {
                 contentEqualsIgnoreCase(contentEncoding, " deflate , foo , gzip "), is(true));
     }
 
-    @ParameterizedTest(name = "{displayName} [{index}] {0}-{1}")
-    @MethodSource("requesterTypes")
-    void contentLengthRemovedOnEncode(final RequesterType type,
-                                      final SecurityType security) throws Exception {
+    @ParameterizedTest(name = "{displayName} [{index}] {0}")
+    @EnumSource(SecurityType.class)
+    void contentLengthRemovedOnEncode(final SecurityType security) throws Exception {
```

Comment:
diff looks ugly but I moved the `AbstractHttpRequesterFilterTest` test to a new class and kept this class focused on service filter.